micro-ui/web/micro-ui-internals/packages/modules/core/src/components/RoleBasedEmployeeHome.js

```
@@ -0,0 +1,172 @@
+  useEffect(() => {
+      setMdmsOrderData(MdmsRes);
+  }, [MdmsRes]);
+
+  const transformURL = (url = "") => {
+    if (url == "/") {
+      return;
+    }
+    if (url?.indexOf(`/${window?.contextPath}`) === -1) {
+      let updatedUrl = null;
+      if (isMultiRootTenant) {
+        url = url.replace("/sandbox-ui/employee", `/sandbox-ui/${tenantId}/employee`);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)